x/gov/client/cli/query.go

```diff
@@ -261,8 +281,9 @@ func GetCmdQueryDeposit(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		Args:  cobra.ExactArgs(2),
 		Short: "Query details of a deposit",
 		Long: strings.TrimSpace(`
-Query details for a single proposal deposit on a proposal. You can find the proposal-id by running gaiacli query gov proposals:
+Query details for a single proposal deposit on a proposal by its identifier.
```

Comment:
_grudging_ this is a better way to say this. Thank you for going through and fixing my bad verbiage here.

Reply:
My pleasure ^__^